lib/src/transaction_builder.dart

```diff
@@ -28,10 +28,8 @@ class TransactionBuilder {
 
   List<Input> get inputs => _inputs;
 
-  factory TransactionBuilder.fromTransaction(
-    Transaction transaction, [NetworkType? network]
-  ) {
-
+  factory TransactionBuilder.fromTransaction(Transaction transaction,
+      [NetworkType? network]) {
```

Comment:
It looks cleaner having the arguments on the same line like before in my opinion.